blocks/editable/index.js

```diff
@@ -612,6 +621,12 @@ export default class Editable extends Component {
 		this.editor.setDirty( true );
 	}
 
+	getToolbarCustomControls() {
```

Comment:
What's the reasoning for moving this filter to `Editable`? Since there's a filter already done in the formatting toolbar, I'd have expected it to be done there?